<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_4">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>267</width>
          <height>228</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_7">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_8">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_9">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_10">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_6">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_5">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <attribute name="label">
         <string>跑</string>
        </attribute>
        <widget class="QPushButton" name="pushButton_11">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>80</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>88</width>
          <height>16</height>
         </rect>
        </property>
        <attribute name="label">
         <string>走</string>
        </attribute>
        <widget class="QToolButton" name="toolButton">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>40</y>
           <width>21</width>
           <height>19</height>
          </rect>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>30</height>
         </rect>
        </property>
        <attribute name="label">
         <string>润</string>
        </attribute>
        <widget class="QRadioButton" name="radioButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>73</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_6">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>日志</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>记录</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>实验</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtoncao1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>scoreup</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtoncao2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>scoredown</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtontext1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>窗口1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtontext2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>窗口2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtoncomeback">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>回到青岛</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelpic1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>pic1</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelgif">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>gif1</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <zorder>labelpic1</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>pushButtoncao1</zorder>
  <zorder>pushButtoncao2</zorder>
  <zorder>spinBox</zorder>
  <zorder>pushButtontext1</zorder>
  <zorder>pushButtontext2</zorder>
  <zorder>comboBox</zorder>
  <zorder>pushButtoncomeback</zorder>
  <zorder>labelgif</zorder>
  <zorder>progressBar</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtoncao1</sender>
   <signal>clicked()</signal>
   <receiver>spinBox</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtoncao2</sender>
   <signal>clicked()</signal>
   <receiver>spinBox</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stackedWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
